requirements/bddtests.txt

```diff
@@ -385,4 +385,14 @@ zope-sqlalchemy==1.6
     # via -r requirements/requirements.txt
 
 # The following packages are considered to be unsafe in a requirements file:
-# setuptools
+setuptools==62.6.0
+    # via
+    #   -r requirements/requirements.txt
+    #   google-auth
+    #   gunicorn
+    #   jsonschema
+    #   plaster
+    #   pyramid
+    #   zope-deprecation
+    #   zope-interface
+    #   zope-sqlalchemy
```

Comment:
The `pip-compile --allow-unsafe` option adds this block to every requirements file